code/modules/awaymissions/maploader/reader.dm

```diff
@@ -255,186 +243,181 @@ GLOBAL_DATUM_INIT(_preloader, /dmm_suite/preloader, new())
 
 	var/first_turf_index = 1
-	while(!ispath(members[first_turf_index],/turf)) //find first /turf object in members
+	while(!ispath(members[first_turf_index], /turf)) // find first /turf object in members
 		first_turf_index++
 
-	//instanciate the first /turf
+	// instanciate the first /turf
 	var/turf/T
 	if(members[first_turf_index] != /turf/template_noop)
-		T = instance_atom(members[first_turf_index],members_attributes[first_turf_index],xcrd,ycrd,zcrd)
+		T = instance_atom(members[first_turf_index], members_attributes[first_turf_index], xcrd, ycrd, zcrd)
 
 	if(T)
-		//if others /turf are presents, simulates the underlays piling effect
+		// if others /turf are presents, simulates the underlays piling effect
 		index = first_turf_index + 1
 		while(index <= members.len - 1) // Last item is an /area
 			var/underlay
 			if(istype(T, /turf)) // I blame this on the stupid clown who coded the BYOND map editor
 				underlay = T.appearance
-			T = instance_atom(members[index],members_attributes[index],xcrd,ycrd,zcrd)//instance new turf
-			if(ispath(members[index],/turf))
+			T = instance_atom(members[index], members_attributes[index], xcrd, ycrd, zcrd) // instance new turf
+			if(ispath(members[index], /turf))
 				T.underlays += underlay
 
 			index++
 
-	//finally instance all remainings objects/mobs
-	for(index in 1 to first_turf_index-1)
-		instance_atom(members[index],members_attributes[index],xcrd,ycrd,zcrd)
+	// finally instance all remainings objects/mobs
+	for(index in 1 to first_turf_index - 1)
+		instance_atom(members[index], members_attributes[index], xcrd, ycrd, zcrd)
 		CHECK_TICK
 
 ////////////////
-//Helpers procs
+// Helpers procs
 ////////////////
 
-//Instance an atom at (x,y,z) and gives it the variables in attributes
-/dmm_suite/proc/instance_atom(path,list/attributes, x, y, z)
+// Instance an atom at (x, y, z) and gives it the variables in attributes
+/datum/dmm_suite/proc/instance_atom(path, list/attributes, x = 0, y = 0, z = 0)
 	var/atom/instance
 	GLOB._preloader.setup(attributes, path)
 
-	var/turf/T = locate(x,y,z)
+	var/turf/T = locate(x, y, z)
 	if(T)
 		if(ispath(path, /turf))
 			T.ChangeTurf(path, defer_change = TRUE, keep_icon = FALSE)
 			instance = T
 		else if(ispath(path, /area))
 
 		else
-			instance = new path (T)//first preloader pass
+			instance = new path (T) // first preloader pass
 
-	if(GLOB.use_preloader && instance)//second preloader pass, for those atoms that don't ..() in New()
+	if(GLOB.use_preloader && instance) // second preloader pass, for those atoms that don't ..() in New()
 		GLOB._preloader.load(instance)
 
 	return instance
 
-//text trimming (both directions) helper proc
-//optionally removes quotes before and after the text (for variable name)
-/dmm_suite/proc/trim_text(what as text,trim_quotes=0)
+// text trimming (both directions) helper proc
+// optionally removes quotes before and after the text (for variable name)
+/datum/dmm_suite/proc/trim_text(what = "", trim_quotes = FALSE)
```

Comment:
The `what` should not be defaulted either. Calling this empty doesn't make much sense